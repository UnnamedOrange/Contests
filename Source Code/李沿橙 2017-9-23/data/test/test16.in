5
6
40 105 18 787 373 606 
50
468 63 882 696 546 677 97 344 56 6 964 358 165 105 22 917 131 29 992 546 561 302 420 828 312 956 978 54 169 467 851 317 947 801 104 123 356 85 261 340 427 233 101 897 726 558 423 172 919 983 
50
363 8 197 997 989 137 82 87 948 152 947 4 114 116 224 188 985 353 960 981 943 126 107 160 360 338 299 183 300 1000 993 128 322 130 248 44 2 996 962 368 235 116 982 940 303 998 989 967 984 953 
44
892 979 996 87 65 23 961 926 852 866 237 130 852 831 859 958 4 61 973 835 146 994 926 239 32 918 34 937 846 196 920 960 842 964 231 79 830 952 855 99 945 877 914 860 
15
3 42 20 992 39 33 23 19 42 13 31 22 14 25 31 
