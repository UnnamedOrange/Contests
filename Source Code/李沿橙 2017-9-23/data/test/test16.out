18 40 105 606 787 373 
6 22 29 54 56 63 85 97 101 104 105 123 165 169 172 233 302 312 317 340 344 356 420 423 427 467 546 546 558 677 726 801 851 882 897 917 919 947 964 978 983 992 956 828 696 561 468 358 261 131 
2 4 8 44 82 87 107 114 116 116 126 128 130 137 152 160 183 188 197 224 235 248 299 300 303 322 338 353 360 363 940 947 948 953 960 962 967 981 982 984 985 989 989 993 996 997 998 1000 943 368 
4 23 32 34 61 65 79 87 99 130 146 196 231 237 830 835 842 846 852 852 855 859 860 866 877 892 914 918 920 926 926 937 945 952 958 960 961 964 973 979 994 996 831 239 
3 13 14 19 20 22 23 25 31 31 33 39 42 992 42 
